src/main/java/Event.java

```diff
@@ -0,0 +1,19 @@
+public class Event extends Task {
+    private String from;
+    private String to;
+
+    public Event(String description, String from, String to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+
+    @Override
+    public String toString() {
+        return "[E][" + getStatusIcon() + "] " + description + " (from: " + from + " to: " + to + ")";
```

Comment:
Consider adding the description in the parent classes toFIleString() to make full use of the inheritance principles